packages/web-frontend/src/containers/MeasureBar/Control.js

```diff
@@ -183,6 +188,10 @@ Control.propTypes = {
     startDate: PropTypes.shape({}),
     endDate: PropTypes.shape({}),
   }),
+  defaultDates: PropTypes.shape({
+    startDate: PropTypes.string,
+    endDate: PropTypes.string,
+  }).isRequired,
```

Comment:
is default date required?

Reply:
Yeah for map overlays we are saying it is required so that we always have initial dates, and so DateRangePicker is guaranteed to use the correct default dates.

Reply:
Added a comment to clarify